nmpolicy/internal/capture/resolver_test.go

```diff
@@ -45,29 +42,29 @@ func TestBasicPolicy(t *testing.T) {
 
 func testNoExpressions(t *testing.T) {
 	t.Run("resolve with no expression", func(t *testing.T) {
-		capCtrl := capture.New(lexerStub{}, parserStub{}, resolverStub{})
-		resolvedCaps, err := capCtrl.Resolve(
+		captureResolver := capture.NewResolver(lexerStub{}, parserStub{}, resolverStub{})
+		resolvedCapture, err := captureResolver.Resolve(
```

Comment:
It is `resolvedCaptures` using a singular form on something that represents multiple resolved captures is confusing.

Reply:
The proper name is `resolvedCaptureExpressions`